src/include/migraphx/operators.hpp

```diff
@@ -62,6 +62,7 @@
 #include <migraphx/op/quant_convolution.hpp>
 #include <migraphx/op/quant_dot.hpp>
 #include <migraphx/op/pow.hpp>
+#include <migraphx/op/prefix_scan_sum.hpp>
```

Comment:
we can move the file name above a little, so they are in alphabetical order. The "pow.hpp" should also be moved